identifiers/country-py/README.md

```diff
@@ -0,0 +1,3 @@
+This folder lists the following [divisions of Paraguay](https://en.wikipedia.org/wiki/Departments_of_Paraguay): 
```

Comment:
Consider adding a note that the identifiers for `capital` and `department` use iso 3166-2 codes

Reply:
Just added this info